Quaver.Shared/Screens/Result/UI/ResultScoreContainer.cs

```diff
@@ -69,6 +73,11 @@ public class ResultScoreContainer : Sprite
         /// </summary>
         public Sprite ResultKeyValueItemDividerLine { get; private set; }
 
+        /// <summary>
+        ///     The horizontal divider line under the statistics key/value items.
+        /// </summary>
+        public Sprite StatisticsKeyValueItemDividerLine { get; private set; }
```

Comment:
Instead of having this, just extend `ResultKeyValueItemDividerLine` to the full width